horovod/tensorflow/elastic.py

```diff
@@ -25,7 +25,7 @@
 from horovod.tensorflow.mpi_ops import _executing_eagerly, init, rank, shutdown
 
 
-_IS_TF2 = LooseVersion(tf.__version__) >= LooseVersion('2.0.0')
+_IS_TF2 = parse(tf.__version__) >= parse('2.0.0')
```

Comment:
I'd prefer if the import line was adapted slightly so this would read `version.parse(...` here. `parse` is pretty generic.